src/core/messages/store/getters.ts

```diff
@@ -0,0 +1,10 @@
+import { GetterTree } from 'vuex';
+
+import { RootStateInterface } from '@/store/types';
+import { MessagesStateInterface } from '@/core/messages/store/types';
+
+export const getters: GetterTree<MessagesStateInterface, RootStateInterface> = {
+  getMessages: (state) => {
+    return state.messages;
```

Comment:
Why do you need this getter if you can work with store directly to get this messages array?